### ISSUE TITLE
Blockers to publishing to crates.io

### ISSUE THREAD

**Issue by c-nixon** [srijs/rust-aws-lambda#31]
Not sure where to put this, feel free to close the issue if there's a better place.

Is there an envisioned state you'd like the repo to get to before publishing the crates to crates.io?

I've been using it for a month of so now and havn't run into any issues. I'd be happy to contribute if there was a list of things that needed tidied up before pushing up a 0.1.

**Comment by srijs** [comment]
I think in terms of maturity we're probably good enough for a 0.1 at this point.

There are blockers with regards to some dependencies of the runtime package, where we currently depend on the following unplublished crates:

```
gob = { git = "https://github.com/srijs/rust-gob" }
...
serde_schema = { git = "https://github.com/srijs/rust-serde-schema", features = ["bytes"] }
serde_schema_derive = { git = "https://github.com/srijs/rust-serde-schema" }
```

It's probably on me to make sure these crates get published. I'll try to get that done over the weekend.

**Comment by LegNeato** [comment]
I'll have some time in the next week or so to just work on open source stuff...feel free to throw tasks my way in any of those projects.

**Comment by srijs** [comment]
As of 9fcd066ffafa6e75bda599adb144cb56dc4ad5a0, the runtime now depends on crates.io versions of these dependencies instead, so this particular blocker is now removed.

**Comment by srijs** [comment]
The bad news is that we should probably resolve #36 as well before we publish the events crate.

**Comment by LegNeato** [comment]
I fixed #36 and published the events crate, as it was the lowest in the chain and is useful for other projects: https://crates.io/crates/aws_lambda_events

**Comment by srijs** [comment]
As we're deprecating this project in favour of https://github.com/awslabs/aws-lambda-rust-runtime, I'm going to close this issue.
